db/schema.rb

```diff
@@ -40,7 +40,9 @@
     t.datetime "updated_at", null: false
     t.bigint "like_id"
     t.string "image"
+    t.bigint "user_id"
     t.index ["like_id"], name: "index_posts_on_like_id"
+    t.index ["user_id"], name: "index_posts_on_user_id"
   end
```

Comment:
I think so good question. Column `user_id` needs to exits in the `posts` table in order for the index between `posts` and `users` to be created through it.